xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminUsersSheet.xml

```diff
@@ -244,7 +244,10 @@
 #macro (maybeShowDeleteUserWarning $userReference $right)
   #countPagesLastModifiedBy($userReference)
   #if ($pageCount &gt; 0)
-    &lt;div class="box errormessage xform"&gt;
+    {{/html}}
+    
+    {{error cssClass="xform"}}
+      {{html clean="false"}}
```

Comment:
Hmm and why do we need the clean false here?

Reply:
Right before starting the velocity macro, we closed the "current" html macro.
The only use of `#maybeShowDeleteUserWarning` is a few lines above, in `#deleteUserModalContent`. In this macro, we can see that the opening HTML block has the parameter `clean="false"`. Since I do not want to break anything, the safest bet is to avoid cleaning the HTML in the content of the error XWiki macro too (It was already in a noncleaned HTML macro before).

Reply:
Note that another potential reason to use clean="false" is performance, since the cleanup can be expensive.

Reply:
Honestly, I don't have a string opinion on either side. I guess it depends on how complex the content is is and how often it's used.

Reply:
| File using `clean="false"` or `clean=false` | position                    | Modified                               | Reason Step 1                                                                                                                                |
| ------------------------------------------- | --------------------------- | -------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- | 
| AdminUsersSheet.xml                         | inside macro content        | Yes (no reason found)<br>$${\color{red}TODO}$$          | --                                                                                                                                           | 
| AdminUsersSheet.xml                         | after macro content         | No                                     | backward compatibility of the `maybeShowDeleteUserWarning` velocimacro.                                                                      | --            |
| LiveTableEditSheet.xml                      | inside macro content        | Yes (no reason found)<br>$${\color{red}TODO}$$          | --                                                                                                                                           |
| CreateWiki.xml                              | before macro content ( x2 ) | No                                     | We don't create HTML that is correct inside this macro alone (elements are opened but closed only in another HTML macro).                    | 
| CreateWiki.xml                              | after macro content ( x2 )  | No                                     | We don't create HTML that is correct inside this macro alone (elements are not opened but closed only here).                                 | 
| InvitationCommon.xml                        | inside macro content        | Yes, remove the HTML block<br>$${\color{lightgreen}Done}$$     | This is not even HTML syntax (with standard values of translations)                                                                          |
| InvitationCommon.xml                        | after macro content         | Yes, but just add `wiki=false`<br>$${\color{lightgreen}Done}$$  | We want to set the same HTML macro context as before the macro (see L133). In order to do so, we should have both wiki=false and clean=false | 
| ExtensionBreakingQuestion.form.vm           | before macro content        | $${\color{lightgreen}Done}$$                                    | We don't create HTML that is correct inside this macro alone (elements are opened but closed only in another HTML macro).                    | 
| ExtensionBreakingQuestion.form.vm           | after macro content         | $${\color{lightgreen}Done}$$                                    | We don't create HTML that is correct inside this macro alone (elements are not opened but closed only here)                                  |
| ExtensionBreakingQuestion.form.vm           | inside macro content        | $${\color{lightgreen}Done}$$                                    | --                                                                                                                                           | 
| XClassBreakingQuestion.form.vm              | before macro content        | $${\color{lightgreen}Done}$$                                    | We don't create HTML that is correct inside this macro alone (elements are opened but closed only in another HTML macro).                    | 
| XClassBreakingQuestion.form.vm              | after macro content         | $${\color{lightgreen}Done}$$                                    | We don't create HTML that is correct inside this macro alone (elements are not opened but closed only here)                                  | 
| XClassBreakingQuestion.form.vm              | inside macro content        | $${\color{lightgreen}Done}$$                                    | --                                                                                                                                           |